src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/AppointmentService.kt

```diff
@@ -103,7 +105,17 @@ class AppointmentService(
 
     appointment.appointmentFeedbackSubmittedAt = OffsetDateTime.now()
     appointment.appointmentFeedbackSubmittedBy = authUserRepository.save(submitter)
-    return appointmentRepository.save(appointment)
+    appointmentRepository.save(appointment)
+
+    appointmentEventPublisher.attendanceRecordedEvent(appointment, appointment.attended!! == Attended.NO, appointmentType)
+
+    if (appointment.attendanceBehaviourSubmittedAt != null) { // excluding the case of non attendance
+      appointmentEventPublisher.behaviourRecordedEvent(appointment, appointment.notifyPPOfAttendanceBehaviour!!, appointmentType)
+    }
+
```

Comment:
So we will have three emails occurring at the same time when the SP submits the feedback?
(1 email for feedback submitted, 1 email for attendance recorded, 1 email for behaviour recorded).
Do the PP users need all of these emails rather than just 1 email for feedback submitted?

Reply:
On that note... should notifyPP not be based on the field notifyPPOfAttendanceBehaviour?